x-pack/plugins/alerting/common/alert_schema/index.ts

```diff
@@ -7,3 +7,5 @@
 
 export { mappingFromFieldMap } from './field_maps/mapping_from_field_map';
 export { getComponentTemplateFromFieldMap } from './field_maps/component_template_from_field_map';
+export { AlertSchema } from './schemas/alert_schema';
+export type { Alert } from './schemas/alert_schema';
```

Comment:
The `Alert` type is auto-generated from the field map and reflects the fields that the framework will populate in the alerts as data documents.